### ISSUE TITLE
Blackjack dando erro

### ISSUE THREAD

**Issue by vfxbeter** [falcao-g/Falbot#4]

**Comment by vfxbeter** [comment]
Opa falcão, acho que você já deve saber disso mas, o blackjack tá dando erro na parte de empate, dando o dobro de dinheiro do que você deu, só passando aqui pra dar um feedback mesmo. de resto o bot está perfeito, parabens pelo trabalho!

**Comment by falcao-g** [comment]
olá, obrigado pelo feedback! Sempre bom quando as pessoas contribuem, esse bug realmente tinha passado despercebido por mim. Falando dele, eu tava fazendo uns testes aqui e parece que com as mudanças que eu fiz no comando recentemente ele foi consertado, mas por enquanto no bot principal ainda vai estar errado então vou deixar a issue aberta até a próxima release.

**Comment by vfxbeter** [comment]
certo, espero que ele seja consertado na próxima realese, tanto que eu virei o mais rico na classificação global só com esse bug (ainda não tinha percebido) mas, no geral, parece que o dinheiro se multiplica x3, não somando apenas o valor que você colocou. enfim, obrigado pela resposta sobre o erro! parabéns pelo trabalho!

**Comment by vfxbeter** [comment]
![image](https://user-images.githubusercontent.com/88866205/223846537-9c7ce2df-8407-4958-9111-63d7e49b4eec.png)
ah, e quando empata, ele não diminui o dinheiro que você tirou, ele apenas soma a quantidade que você colocou, então, com o empate você ganha falcoins

**Comment by vfxbeter** [comment]
de nada! acabei de testar, tudo funcionando! qualquer coisa se precisar testar, estou a disposição!
